content/influxdb/v2.4/reference/release-notes/influx-cli.md

```diff
@@ -8,6 +8,23 @@ menu:
     name: influx CLI 
 ---
 
+## v2.5.0 [2022-10-20]
+
+### Features
+
+- Support username and password in [influx auth create](/influxdb/v2.4/reference/cli/influx/auth/create/).
```

Comment:
@jeffreyssmith2nd did we add new flags for `username` and `password` to `influx auth create` or `influx config set` or does this mean the existing flags `--user` and `--token` are now working properly?

Reply:
`influx config update` and `influx config create` both have a `--username-password` flag now. This flag takes either `username:password` or just `username`. If a user provides both, then all their requests using that config will automatically be session auth'd. If they just provide username, each command execution will prompt them for their password and then create the session.

Reply:
thank you, @jeffreyssmith2nd, were there any changes to influx auth create...or is that just working with the parameters listed out there now? Curious bc that's where we added the "workaround"

Reply:
@jeffreyssmith2nd to be sure, by `influx config update` did you mean `influx config set`?

Reply:
`update` and `set` are aliases, they're the same command under the hood. Feel free to refer to it however docs normally do, I just happen to use `update` more so that's what I used here.